install.html

```diff
@@ -38,7 +38,7 @@
       <p>Grab Codeception phar executable</p>
       <div>
         <p>
-          <pre><code>wget http://codeception.com/codecept.phar .</code></pre>
+          <pre><code>wget http://codeception.com/codecept.phar</code></pre>
```

Comment:
many machines do not have wget, is also an option that we say here:

```
curl -O http://codeception.com/codecept.phar
```

related http://www.thegeekstuff.com/2012/07/wget-curl/